src/frontend/components/Button/index.spec.tsx

```diff
@@ -8,9 +8,9 @@ describe('<Button />', () => {
     render(<Button label="Button" />);
 
     screen.getByRole('button');
-    screen.getByRole('button_title');
-    expect(screen.queryByRole('icon_container')).not.toBeInTheDocument();
-    expect(screen.queryByRole('icon_resizer')).not.toBeInTheDocument();
+    screen.getByTestId('button_title');
```

Comment:
Almost all `screen.getByTestId` in this file can be replaced by `getByText`. For the icon you are already using `getByText` for exemple.

Reply:
2e52b75